src/Step/StepModel/RomanIntSteps.php

```diff
@@ -139,12 +140,12 @@ public function getRomanToIntSteps(string $roman): array
 
 
 	/**
-	 * @param string $int (as integer)
+	 * @param string $input (as integer)
 	 * @return Step[]
 	 */
-	public function getIntToRomanSteps(string $int): array
+	public function getIntToRomanSteps(string $input): array
 	{
-		$input = $int;
+		$int = (int) $input;
```

Comment:
Retype to `int` is not safe for big numbers. I think `string` representation is better.

Reply:
However, you do integer operations with the variable, so it was cast in the past. I'll fix it.

Reply:
Sorry, the variable has been marked as a string to ensure forward compatibility.

The intention is to gradually convert all functions to bcmath.

Reply:
Fixed via brick/math as a best practice